operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -47,6 +47,181 @@ It may fail to identify installed components or vulnerabilities if the final ima
++
+[source,yaml,subs="+quotes"]
+----
+customize:
+  sensor:
+    envVars:
+      *ROX_DELEGATED_SCANNING_DISABLED: "true"*
+----
++
+For more information about setting the environment variables, see xref:../installing/installing_ocp/install-secured-cluster-ocp.adoc#secured-cluster-services-environment-variables_install-secured-cluster-ocp[Environment variables].
+
+
+[id="configuring-environment-variables-after-installation_{context}"]
+==== Configuring environment variables after installation
+
+[discrete]
+===== OpenShift Container Platform
+
+.Procedure
+
+* Run the following command to enable delegated scanning by setting the value of the environment variable`ROX_DELEGATED_SCANNING_DISABLED` to `true` for a deployment named `sensor`:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc set env deploy/sensor ROX_DELEGATED_SCANNING_DISABLED="true"
+----
+
+[discrete]
+===== Kubernetes
+
+.Procedure
+
+* Run the following command to enable delegated scanning by setting the value of the environment variable `ROX_DELEGATED_SCANNING_DISABLED` to `true` for a deployment named `sensor`:
++
+[source,terminal,subs="+quotes"]
+----
+$ kubectl set env deploy/sensor ROX_DELEGATED_SCANNING_DISABLED="true"
+----
+
+Sensor should restart automatically. However, if it doesn't restart on its own, you should manually cause it to restart for the changes to take effect.
+
+If you're using the {product-title-short} Operator, there is a possibility that this change can be undone. In such a case, you should follow the specific instructions of the Operator.
+
+
+[id="configuring-delegated-image-scanning_{context}"]
+=== Configuring delegated image scanning
+
+A new delegated registration configuration specifies the registries from which image queries are to be delegated. For images observed by Sensor, this configuration allows you to delegate scans from no registries (default), all registries, or specific registries. To enable delegation of scans using the `roxctl` CLI, the Jenkins plugin, or the API, you must also specify a destination cluster and source registry.
+
+
+.Prerequisites
+
+* The environment variables must be configured. For more information on how to configure environment variables, see xref:../operating/examine-images-for-vulnerabilities.adoc#configuring-environment-variables_examine-images-for-vulnerabilities[Configuring environment variables].
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Manage delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the necessary information:
+** *Delegate scanning for*: Choose the scope of image delegation. Select one of the following options:
+*** None (default)
+*** All registries
+*** Specified registries
+** *Select default cluster to delegate to*: From the drop-down list, select the name of the destination cluster to which the delegated scans should be sent.
```

Comment:
@dcaravel 
The way this is worded, it sounds like this is required, but you said it's not.

Should we call out that this is optional, but explicitly state the effect of leaving it blank?

Reply:
Yes, this is optional and we should call that out. There has been alot of confusion regarding the cluster IDs, it will be great to explain that here.  Anywhere a cluster id is mentioned we should qualify that its only for CLI/API requests only and is NOT used by sensor.

The [API docs](https://github.com/stackrox/stackrox/blob/1f0c53ab12388e8997ba5b574696b9c9aadfe665/proto/api/v1/delegated_registry_config_service.proto#L22-L58) explain in more detail what each setting does.

May even be worth separating how this feature behaves from a sensor perspective vs. a roxctl/api perspective, for each setting.